### PR TITLE
(no homo) Changing "gay" word in cloning code to "expected amounts"

### PR DESCRIPTION
**Pull request by niceping**
basically copy&paste from https://github.com/SunsetStation/SunsetStation/pull/640

i can't code so if something is wrong help me

:cl: no homo
spellcheck: changed "gay" in cloning code to "expected amounts"
/:cl:

[why]: # original PR maker said it somehow got passed, and if people dont want to get it fixed they can close it anyways

### CODE REVIEW

**Comment by kingoftheautism** [comment]
wait did i accidently did [code] instead of [tweak] or [fix]? oof

**Comment by yoyobatty** [comment]
Shouldn't this be something related to the actual rating of the stock parts

**Comment by kingoftheautism** [comment]
i cant code, if you have options tell me, i will change line again

**Comment by kriskog** [comment]
So currently it outputs: "The status display reads: Speed gay."
It would seem he forgot to actually add a variable (or a function of) here. Replacing that with "Expected amounts" will output: "The status display reads: Speed Expected amounts."
As you can see, that isn't really a fix.

Replace it with: Speed: <b>+[speed_coeff * 12.5]%</b>.
That should make more sense.

**Comment by Iamgoofball** [comment]
this was incredibly immature

**Comment by 81Denton** [comment]
Uhh that's embarassing, that was supposed to be an actual value. Don't drink and code

File: code/game/machinery/cloning.dm [hunk 1]
```diff
@@ -77,7 +77,7 @@
 /obj/machinery/clonepod/examine(mob/user)
 	..()
 	if(in_range(user, src) || isobserver(user))
-		to_chat(user, "<span class='notice'>The status display reads: Speed <b>gay</b>.<br>Predicted amount of cellular damage: <b>[100-heal_level]%</b>.<span>")
+		to_chat(user, "<span class='notice'>The status display reads: Speed <b>Expected amounts</b>.<br>Predicted amount of cellular damage: <b>[100-heal_level]%</b>.<span>")
```
Comment (81Denton) [inline code/game/machinery/cloning.dm]:
```suggestion
		to_chat(user, "<span class='notice'>The status display reads: Cloning speed at <b>[speed_coeff*50]%</b>.<br>Predicted amount of cellular damage: <b>[100-heal_level]%</b>.<span>")
```

Comment (81Denton) [reply code/game/machinery/cloning.dm]: With that, it will show the cloning speed as 100%/200%/300%/400%.

**Comment by 81Denton** [comment]
And yeah sorry about that, normally I review my code before pushing

**Comment by kingoftheautism** [comment]
appveyor build canceled? shit fucked

gonna actually try to fix this without webedits as soon as i will get to my pc

**Comment by MMMiracles** [comment]
how fast is 'gay' speed?

**Comment by Time-Green** [comment]
blue

**Comment by Iamgoofball** [comment]
I'm not sure why my comment was marked as disruptive content when it's literally true.

**Comment by ma44** [comment]
@Iamgoofball https://github.com/tgstation/tgstation/pull/40920/files#diff-1e264196f81ead51bb747ac7d38f1a09R80

**Pull request merged**